frontend/src/components/pages/teacher/DistributeAssessmentPage.tsx

```diff
@@ -109,6 +123,16 @@ const DistributeAssessmentPage = (): React.ReactElement => {
             Next
           </Button>
         )}
+        {page === 3 && (
+          <Button
+            leftIcon={<PaperPlaneOutlineIcon />}
+            minWidth="10"
+            onClick={() => console.log("Distribute assessment.")}
+            variant="primary"
+          >
+            Distribute
+          </Button>
+        )}
```

Comment:
I moved the location of the button, I'll check with design that it's okay.

Reply:
kinda like this more for consistency